src/components/forms/checkbox/Checkbox.vue

```diff
@@ -0,0 +1,233 @@
+<script lang="ts" setup>
+import { type ContextColorsType } from '@/consts/colors';
+import Icon from '@/components/icons/Icon.vue';
+
+const props = withDefaults(
+  defineProps<{
+    value?: boolean;
```

Comment:
Since we are target vue-3 for v-model shouldn't this be `modelValue` and the emitter `update:modelValue`?